### ISSUE TITLE
Could you add serch by RegExp?

### ISSUE THREAD

**Issue by MrHIDEn** [wayou/vscode-todo-highlight#133]
Could you add serch by RegExp like:
"text": /\/\/ *TODO.*/

**Comment by stellarbeam** [comment]
Yes, via the `todohighlight.keywordsPattern` option, you can specify a RegExp. 
Note that `todohighlights.keywords` will be ignored if `todohighlight.keywordsPattern` is set.
Refer the README's example configuration for more info.
